### DISCUSSION TITLE
Member "withdrawFundMe" not found or not visible after argument-dependent lookup in contract WithdrawFundMe.solidity(9582)

### DISCUSSION THREAD

**Discussion by king-null** [Cyfrin/foundry-full-course-cu#1520] [Q&A]
i encountered this  error in my testUserCanFundInteractions function inside Interactions.t.sol  i saw this error title when i hover in the withdrawFundMe.withdrawFundMe(address(fundMe)); , i have also checked my WithdrawFundMe contract and withdrawFundMe function also it seems to be in order so i dont know where the error is coming from

this below is my WithdrawFundMe contract

```solidity
contract WithdrawFundMe is Script {
    function withdrawFundMe(address mostRecentlyDeployed) public {
        FundMe(payable(mostRecentlyDeployed)).withdraw();
    }

    function run() external {
        address mostRecentlyDeployed = DevOpsTools.get_most_recent_deployment(
            "FundMe",
            block.chainid
        );
        vm.startBroadcast();
        withdrawFundMe(mostRecentlyDeployed);
        vm.stopBroadcast();
    }

```

**Comment by EngrPips** [comment]
Hello @king-null, I  must confess that i would need more details and insight on what is going on to be able to debug your issue , maybe provide a screenshot of your `interaction.t.sol` file and specifically the `testUserCanFundInteractions` function and also a full screenshot of your `WithdrawFundMe` script. Or better still just push your code to github and share a link with me like that it would very easy for me to help you debug

**Reply by king-null** [reply]
hello @EngrPips i dont know what happened but it is fixed all of a sudden, no more error messages so i think everything was right before just my compiling machine misbehaving

**Reply by EngrPips** [reply]
Hey buddy @king-null , i am glad it is fixed now and yeah it is difficult and it took me going through the whole course 4 times to completely understand every single details in the course, so just keep going with time it would get more easy.

**Comment by Niferu** [comment]
Hello @king-null 

There is definitely issue in your tests. You are probably incorrectly deploying contract, or calling incorrect one using wrong address. As @EngrPips mentioned, please share your project on github or try solving it following my ideas

**Reply by king-null** [reply]
yes i agree with you @Niferu , i'm also facing some errors too but i just paused and trying to understand all the code ATM, i must say none of these is easy for a starter like myself, and i appreaciate everyone bringing out there precious time to answer questions here , i hope one day i will be able to answer questions here too , i look forward to that day.
